docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -429,6 +429,38 @@ Result:
 │ {}                         │
 └────────────────────────────┘
 ```  
+
+
+## mapFromArrays {#mapFromArrays}  
+  
+**Syntax**
+
+```sql
+mapFromArrays(keys, values)
+```  
+  
+**Parameters**
+  
+-   `keys` — Given key array to create a map from.
+-   `values`  - Given value array to create a map from.
+  
+**Returned value**
+
+- A map whose keys and values are from input arrays respectively.
```

Comment:
`are from input arrays respectively` --> `are constructed from the key and value arrays`